xmtp_proto/buf.gen.yaml

```diff
@@ -16,6 +16,8 @@ plugins:
       - no_server=true
       - compile_well_known_types
       - extern_path=.google.protobuf=::pbjson_types
+      # Exclude it from non-tonic builds (so we can use the rest in Wasm)
+      - client_mod_attribute=xmtp.message_api.v1=#[cfg(feature = "tonic")]
```

Comment:
This was the sneaky buf change to include a `cfg` that excludes the tonic client from non-tonic builds.

Reply:
Clever